_layouts/download.html

```diff
@@ -175,8 +175,8 @@ <h2>Supported platforms</h2>
 				<span>Linux</span>
 			</a>
 			<a href="/download/macos" class="tab title-font {% if page.platform == 'MacOS' %} active {% endif %}">
-				<img width="24" height="24" src="/assets/icons/macos.svg" title="MacOS" alt="MacOS" />
-				<span>MacOS</span>
+				<img width="24" height="24" src="/assets/icons/macos.svg" title="macOS" alt="macOS" />
+				<span>macOS</span>
```

Comment:
Same should be done to the 3.x downloads page, right?

Reply:
This file seems to be used for both, probably no extra changes required (it's `macOS` since 2016, so relevant for all versions).

Reply:
Yeah, we have dedicated layouts so it's easier to adjust incompatible differences :)